### DISCUSSION TITLE
无法登录 点击发送邮件 转了半天不是6000超时 ，就是403

### DISCUSSION THREAD

**Discussion by txwgoogol** [getlantern/lantern-forum-cn#824] [Q&A]
如题 
邮箱 colin.googol@gmial.com

**Comment by txwgoogol** [comment]
![20240902150901](https://github.com/user-attachments/assets/39d1bb51-cc76-4c15-b0cb-a73597ed8274)
点击一直停留在当前页面，无法使用付费账号

**Comment by txwgoogol** [comment]
![503](https://github.com/user-attachments/assets/38c74403-7859-48c7-913a-850b2e33e171)
HELP！！！

**Comment by MaxMKMax** [comment]
@txwgoogol 您好. 已经给您反馈问题上去了. 非常抱歉影响了您的体验感.
您可以提供我支付单据吗? 谢谢.

**Reply by txwgoogol** [reply]
![WechatIMG34](https://github.com/user-attachments/assets/67a67938-f27a-470c-8c34-143526aa9243)
